.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -3,17 +3,17 @@
 # SPDX-License-Identifier: MPL-2.0+
 # SPDX-FileCopyrightText: 2020-2025 gucci-on-fleek and Voidless7125
 name: Bug Report
-description: Report an issue with “Lockdown Browser in Windows Sandbox”
+description: Report an issue with "Lockdown Browser in Windows Sandbox"
 labels: ["untriaged"]
 body:
   - type: markdown
     attributes:
       value: |
         If this is a general question, please [start a discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new?category=q-a).
 
-        If this is a feature request, please [start a discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new?category=feature-requests) and we'll convert it into an issue once we've triaged it.
+        If this is a feature request, please [start a discussion](https://github.com/gucci-on-fleek/lockdown-browser/discussions/new?category=feature-requests), and we'll convert it into an issue once we've triaged it.
 
-        Otherwise, please provide as much information as possible, and we'll hopefully be able to solve your issue.
+        Otherwise, please provide as much information as possible, and we'll be able to solve your issue.
```

I think leave the “hopefully” in there. We're just offering best-effort support, and some issues are either impossible for us to solve or will take way too much effort to solve.
```suggestion
        Otherwise, please provide as much information as possible, and we'll hopefully be able to solve your issue.
```